defaults/main.yaml

```diff
@@ -1,4 +1,28 @@
 ---
+# Variables from Anarchy Subject
+guid: "{{ vars.anarchy_subject.vars.job_vars.guid | default(None) }}"
+
+# Variables from Anarchy Subject
+uuid: "{{ vars.anarchy_subject.vars.job_vars.uuid | default(None) }}"
+
+# Variables from either Governor or Subject
+env_type: >-
+  {{ vars.anarchy_governor.vars.job_vars.env_type
+  | default(vars.anarchy_subject.vars.job_vars.env_type)
+  | default(None) }}
+
+requester_email: >-
+  {{ vars.anarchy_subject.metadata.annotations['poolboy.gpte.redhat.com/resource-requester-email']
+  | default(vars.anarchy_subject.vars.job_vars.requester_email)
+  | default(vars.anarchy_subject.vars.job_vars.email)
+  | default(None) }}
+
+requester_user: >-
+  {{ vars.anarchy_subject.metadata.annotations['poolboy.gpte.redhat.com/resource-requester-user']
+  | default(vars.anarchy_subject.vars.job_vars.requester_username)
+  | default(vars.anarchy_subject.vars.job_vars.student_name)
+  | default('babylon') }}
```

Comment:
I think perhaps this should be `default('babylon', True)` to force the username "babylon" when it for some reason comes through as empty string?